src/modules/vtol_att_control/vtol_att_control_main.h

```diff
@@ -210,6 +210,7 @@ class VtolAttitudeControl : public ModuleBase<VtolAttitudeControl>, public Modul
 	vehicle_land_detected_s			_land_detected{};
 	vehicle_local_position_s		_local_pos{};
 	vehicle_local_position_setpoint_s	_local_pos_sp{};
+	vehicle_status_s 			_vehicle_status{};
```

Comment:
I decided to store _vehicle_status here because I didn't like to have to call _vehicle_status_sub.copy twice (in both vehicle_status_poll and vehicle_cmd_poll)